File: android/app/src/main/java/com/ms/win32/User32.java
package com.ms.win32;

public class User32 {
    public static int GetWindowLong(int hwnd, int nIndex) {
        return 0;
    }

    public static int SetWindowLong(int hwnd, int nIndex, int dwNewLong) {
        return 0;
    }

    public static int SetWindowLong(int hwnd, int nIndex, Object proc) {
        return 0;
    }

    public static boolean SetWindowPos(int hwnd, int hwndInsertAfter, int x, int y, int cx, int cy, int flags) {
        return true;
    }

    public static int ShowWindow(int hwnd, int nCmdShow) {
        return 0;
    }

    public static void SetCursorPos(int x, int y) {
    }

    public static int LoadCursor(int instance, int cursor) {
        return 0;
    }

    public static int SendMessage(int hwnd, int msg, int wParam, int lParam) {
        return 0;
    }

    public static int SetCursor(int cursor) {
        return 0;
    }

    public static int CallWindowProc(int prev, int hwnd, int msg, int wParam, int lParam) {
        return 0;
    }
}

File: src/commonMain/kotlin/com/ms/win32/User32.kt
package com.ms.win32

import com.ms.dll.Callback

expect object User32 {
    fun SetWindowLong(i: Int, i_0_: Int, i_1_: Int): Int

    fun GetWindowLong(i: Int, i_2_: Int): Int

    fun CallWindowProc(
        callback: Callback?, i: Int, i_3_: Int,
        i_4_: Int, i_5_: Int
    ): Int

    fun SetCursor(i: Int): Int

    fun SetWindowLong(i: Int, i_6_: Int, callback: Callback?): Int

    fun SetCursorPos(i: Int, i_7_: Int): Boolean

    fun LoadCursor(i: Int, i_8_: Int): Int

    fun SendMessage(
        i: Int, i_9_: Int, i_10_: Int,
        `object`: Any?
    ): Int

    fun CallWindowProc(
        i: Int, i_11_: Int, i_12_: Int, i_13_: Int,
        i_14_: Int
    ): Int

    fun SetWindowLong(i: Int, i_15_: Int, `object`: Any?): Int
}